### PR TITLE
Improve documentation for Field.addfield and Field.getfield

### PR DESCRIPTION
**Pull request by doismellburning**
Copied from doc/scapy/build_dissect.rst

### CODE REVIEW

**Comment by doismellburning** [comment]
Travis fails due to a (seemingly unrelated) OpenSSL error under PyPy...

```
Traceback (most recent call last):
  File "<input>", line 2, in <module>
  File "<input>", line 24, in test_tls_server
RuntimeError: OpenSSL returned with error code
ERROR: InvocationError for command '/usr/bin/sudo -E /home/travis/build/secdev/scapy/.tox/pypy-linux_root/bin/python -m coverage run --rcfile=.coveragerc.tox -a -m scapy.tools.UTscapy -c ./test/configs/linux.utsc -K random_weird_py3 -K tshark -K not_pypy' (exited with code 1)
```

File: scapy/fields.py [hunk 1]
```diff
@@ -124,11 +124,22 @@ def i2repr(self, pkt, x):
         return repr(self.i2h(pkt, x))
 
     def addfield(self, pkt, s, val):
-        """Add an internal value  to a string"""
+        """Add an internal value to a string
+
+Copy the network representation of field `val` (belonging to layer `pkt`) to
```
Comment (gpotter2) [inline scapy/fields.py]: Could you keep the indent ?

Comment (doismellburning) [reply scapy/fields.py]: Sure! I was just following convention from `MultipleTypeField` a few lines down

**Comment by doismellburning** [comment]
❤️

**Pull request merged**